src/odeproblem.cpp

```diff
@@ -648,6 +648,12 @@ void odeproblem::copy_parin(const Rcpp::List& parin) {
   ss_fixed = Rcpp::as<bool>(parin["ss_fixed"]);
   Rtol = Rcpp::as<double>(parin["rtol"]);
   Atol = Rcpp::as<double>(parin["atol"]);
+  ssRtol = Rcpp::as<double>(parin["ss_rtol"]);
+  ssAtol = Rcpp::as<double>(parin["ss_atol"]);
+  if(Advan==13) {
```

Comment:
Are `ssRtol` & `ssAtol` used in other ADVAN? If not it's better to set them to easy-to-fail values(0 or negative  or a very large value). So that bugs caused by using them by mistake tend to fail dramatically.

Reply:
Yes; `ssRtol` and `ssAtol` are used for assessing approach to steady state regardless of advan.  What should this get changed to?  Or should we just not fiddle with what the user passes in?

Reply:
Still want you feedback on this one.  I mean would it ever make sense to ask for 1e-10 on the ss side when the solver will only go to 1e-5?  Should there be a warning if this happens?

Reply:
> would it ever make sense to ask for 1e-10 on the ss side when the solver will only go to 1e-5?

No.

> Should there be a warning if this happens?

Yes.

Reply:
> What should this get changed to? Or should we just not fiddle with what the user passes in?

No changed needed now as the code simply works. But in the future it's best `odeproblem` class gets refactored to indicate the usage of ssRtol/ssAtol, possibly in the constructor.

Reply:
I see what you're getting at now.  Will do that.

Reply:
I'll bet this would get picked up by the sanitizer they are running on CRAN too.